data/du.txt

```diff
@@ -0,0 +1,5 @@
+T , 1 , sth
+D , 0 , sth else  , 2021-08-22 18:30
+T , 0 , yay
+T , 0 , hi
+D , 0 ,  , 2021-08-16 16:30
```

Comment:
Should we put the data file in `.gitignore`, because I think each person needs their own data?